task1/TaskMethods.cs

```diff
@@ -0,0 +1,361 @@
+            decimal result = PerformConvertation(amount, currency);
+            Report(amount, result, currency);
+        }
+
+        private static decimal PerformConvertation(decimal amount, string currency)
+        {
+            decimal result = 0.00m;
+            switch (currency)
+            {
+                case "dollar":
+                    result = Convert.ToDecimal(Convert.ToDouble(amount) * 2.777);
+                    break;
+                case "euro":
+                    result = Convert.ToDecimal(Convert.ToDouble(amount) * 3.125);
+                    break;
+                case "pound":
+                    result = Convert.ToDecimal(Convert.ToDouble(amount) * 5.421);
+                    break;
+                default:
+                    break;
+            }
+            return result;
+        }
+
+        private static void Report(decimal amount, decimal result, string currency)
+        {
+            Console.WriteLine($"Amount of money {amount} is {result} {currency}");
+        }
+
+        //
+        // Credit.
+        //
+        public static void GetCreditCondition()
+        {
+            decimal credit = 700;
+            Console.WriteLine("Input amount of payment");
+            Decimal.TryParse(Console.ReadLine(), out decimal amount);
+            if (amount < 100)
+            {
+                Console.WriteLine("Payment must be 100 minimum");
+                return;
+            }
+            decimal rest = CreditCalculation(credit, amount);
+            AnalyzeOfRest(rest, credit);
+        }
+
+        private static void AnalyzeOfRest(decimal rest, decimal credit)
+        {
+            if (rest > 0)
+                Console.WriteLine($"You have to pay {rest}");
+            if (rest < 0)
+                Console.WriteLine($"Overpayment is {Math.Abs(rest)}");
+            if (rest == 0)
+                Console.WriteLine("Credit is paid");
+
+        }
+
+        private static decimal CreditCalculation(decimal credit, decimal amount)
+        {
+            return credit - amount;
+        }
+
+        //
+        // ATM Pin validation.
+        //
+        public static bool Validation()
+        {
+            Console.WriteLine("Input PIN");
+            string inputStr = Console.ReadLine();
+            return ValidatePin(inputStr);
+        }
+
+        private static bool ValidatePin(string inputStr)
+        {
+            if (isAllNumbers(inputStr) && isNumOfElementsValid(inputStr))
+                return true;
+            else
+                return false;
+        }
+
+        private static bool isNumOfElementsValid(string inputStr)
+        {
+            if (inputStr.Length == 4 || inputStr.Length == 6)
+                return true;
+            else
+                return false;
+        }
+
+        private static bool isAllNumbers(string inputStr)
+        {
+            return Int32.TryParse(inputStr, out int result);
```

Comment:
there si also method char.IsNumeric, that helps us to define if the symbol in string is numeric.